cmake/Options.cmake

```diff
@@ -10,12 +10,18 @@ else()
   set(LINUX OFF)
 endif()
 
+include(CMakeDependentOption)
+
+# TODO: Default to ON if CUDA available.
```

Comment:
Doesn't this TODO just boil down to doing `find_package(CUDA QUIET)` and then making the default be `CUDA_FOUND`?

Reply:
Yes that's it. I'll add it in a separate PR.